README.md

```diff
@@ -51,6 +55,10 @@ Note: Hash should be in lowercase in index.json otherwise CI will fail.
 
 ### How to fill in the metadata for an index.json entry?
 
+The `azdev extension update-index` command will add the appropriate metadata to the index.json file. For more information visit https://github.com/Azure/azure-cli-dev-tools.
```

Comment:
Can you add some info about how `update-index` will actually add a new entry? Right now this sounds like, if I had the entry for the new version of an extension, a `metadata` field will be added with the right info to that entry.

Reply:
Updated the wording. Let me know what you think. I'm trying to avoid putting too many implementation details here to avoid the readme going stale as the command evolves.